slack-base/src/test/java/com/hubspot/slack/client/models/migration/MigrationExchangeParamsDeserializationTest.java

```diff
@@ -0,0 +1,64 @@
+package com.hubspot.slack.client.models.migration;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Test;
+
+import com.hubspot.slack.client.jackson.ObjectMapperUtils;
+import com.hubspot.slack.client.models.JsonLoader;
+import com.hubspot.slack.client.models.response.migration.MigrationExchangeResponse;
+
+public class MigrationExchangeParamsDeserializationTest {
```

Comment:
The first three comments in this file depend on the last one.

Reply:
Also, you'll need to use `import static org.assertj.core.api.Assertions.assertThat;` in order to change the assertions as I suggested.

Reply:
Thank you!